apis/r/NEWS.md

```diff
@@ -10,6 +10,7 @@
 * Add support for resume-mode in `write_soma()`
 * Push default-setting for `TileDBCreateOptions` to `$initialize()` instead of in the accessors
 * Muffle warnings for missing command logs when outgesting SOMA to `Seurat`
+* Ensure `SOMADataFrame$shape()` always returns `NA`
```

Comment:
That now needs an update as it errors

Reply:
Updated in [`fc9e5d7`](https://github.com/single-cell-data/TileDB-SOMA/pull/2531/commits/fc9e5d7ef91445f92a5596d8167ecc24edbb88d5)